DOCSP-40414: jvm crypto improvement (#86)
10gen/docs-shared: dbx/jvm/v5.2-wn-items.rst

  can use a configured FIPS-compliant JCA provider to provide a higher
  level of security.

- Performance improvements due to implementation of native cryptography
  on all supported platforms. The following list describes the actions
  needed to implement this improvement depending on your operating
  system:
  

>   - **Windows**: Upgrade to your ``mongodb-crypt`` version to v5.2.0.

Reviewer: I: assuming this should be like this?

```suggestion
  - **Windows**: Upgrade your ``mongodb-crypt`` version to v5.2.0.
```

>   - **Mac**: Upgrade to your ``mongodb-crypt`` version to v5.2.0.

Reviewer: ```suggestion
  - **Mac**: Upgrade your ``mongodb-crypt`` version to v5.2.0.
```

  - **Linux**: Install ``libmongocrypt.so`` directly on the file system,
    instead of using the file that is bundled within the

>     ``mongodb-crypt`` JAR file. View Linux instructions for installing

Reviewer: S: maybe a little less direct

```suggestion
    ``mongodb-crypt`` JAR file. You can find Linux instructions for installing
```

    ``libmongocrypt`` in the :manual:`Server manual
    </core/csfle/reference/libmongocrypt/#linux-installation>`. If you

>     use a package manager to install ``libmongocrypt``, JNA will find it

Reviewer: Q: Has JNA already been defined on this page/these pages?

Author: wrote it out

    there without further configuration. If there is an issue, you can

>     specify the search path by setting ``LD_LIBRARY_PATH``.

Author: I will tag tech reviewer to answer this

>     specify the search path by setting ``LD_LIBRARY_PATH``.

Author: @jyemin

>     specify the search path by setting ``LD_LIBRARY_PATH``.

Reviewer 2: I would just change "If there is an issue" to: "Alternatively"
